test/powershell/Modules/Microsoft.PowerShell.Utility/Default-Aliases.Tests.ps1

```diff
@@ -17,3 +20,185 @@ Describe "Windows aliases do not conflict with Linux commands" -Tags "CI" {
+            "rwmi -> Remove-WmiObject"                = $FullCLR
+            "sajb -> Start-Job"                       = $FullCLR -or $CoreWindows -or $CoreUnix
+            "sal -> Set-Alias"                        = $FullCLR -or $CoreWindows -or $CoreUnix
+            "saps -> Start-Process"                   = $FullCLR -or $CoreWindows -or $CoreUnix
+            "sasv -> Start-Service"                   = $FullCLR -or $CoreWindows -or $CoreUnix
+            "sbp -> Set-PSBreakpoint"                 = $FullCLR -or $CoreWindows -or $CoreUnix
+            "sc -> Set-Content"                       = $FullCLR -or $CoreWindows -or $CoreUnix
+            "scb -> Set-Clipboard"                    = $FullCLR
+            "select -> Select-Object"                 = $FullCLR -or $CoreWindows -or $CoreUnix
+            "set -> Set-Variable"                     = $FullCLR -or $CoreWindows -or $CoreUnix
+            "shcm -> Show-Command"                    = $FullCLR
+            "si -> Set-Item"                          = $FullCLR -or $CoreWindows -or $CoreUnix
+            "sl -> Set-Location"                      = $FullCLR -or $CoreWindows -or $CoreUnix
+            "sleep -> Start-Sleep"                    = $FullCLR -or $CoreWindows
+            "sls -> Select-String"                    = $FullCLR -or $CoreWindows -or $CoreUnix
+            "sort -> Sort-Object"                     = $FullCLR -or $CoreWindows
+            "sp -> Set-ItemProperty"                  = $FullCLR -or $CoreWindows -or $CoreUnix
+            "spjb -> Stop-Job"                        = $FullCLR -or $CoreWindows -or $CoreUnix
+            "spps -> Stop-Process"                    = $FullCLR -or $CoreWindows -or $CoreUnix
+            "spsv -> Stop-Service"                    = $FullCLR -or $CoreWindows -or $CoreUnix
+            "start -> Start-Process"                  = $FullCLR -or $CoreWindows
+            "stz -> Set-TimeZone"                     = $CoreWindows
+            "sujb -> Suspend-Job"                     = $FullCLR
+            "sv -> Set-Variable"                      = $FullCLR -or $CoreWindows -or $CoreUnix
+            "swmi -> Set-WmiInstance"                 = $FullCLR
+            "tee -> Tee-Object"                       = $FullCLR -or $CoreWindows
+            "trcm -> Trace-Command"                   = $FullCLR
+            "type -> Get-Content"                     = $FullCLR -or $CoreWindows -or $CoreUnix
+            "wget -> Invoke-WebRequest"               = $FullCLR
+            "where -> Where-Object"                   = $FullCLR -or $CoreWindows -or $CoreUnix
+            "wjb -> Wait-Job"                         = $FullCLR -or $CoreWindows -or $CoreUnix
+            "write -> Write-Output"                   = $FullCLR -or $CoreWindows
+        }
+        $aliaslist = @()
+        $aliaslist += $aliasFullList.Keys | ForEach-Object { if ($aliasFullList[$_]) { $_ } }
```

Comment:
Don't use `+=` on arrays, you can rewrite this as:
```powershell
$aliaslist = @($aliasFullList.Keys | ForEach-Object { if ($aliasFullList[$_]) { $_ } })
```
and it will create many fewer objects to be garbage collected and copy much less.